src/main/java/com/mparticle/kits/GoogleAnalyticsFirebaseKit.java

```diff
@@ -170,12 +170,6 @@ public String getName() {
                     instance.logEvent(eventName, lBundle);
                 }
                 break;
-            case Product.REMOVE_FROM_WISHLIST:
```

Comment:
did we mean to remove this? I didn't see anything mentioned in the ticket or the summary about this, or maybe I'm missing it..